specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/WebApps.json

```diff
@@ -3299,29 +3317,31 @@
             }
           }
         }
-      },
-      "delete": {
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/functions/{functionName}/properties/state": {
```

Comment:
Why is state exposed as an API rather than a property on the resource?

Reply:
There is a new property "isDisabled" that is added to the Function resource as part of this PR. It reflects this state. However, that property isn't writable and doesn't support PATCH updates. Looking back, we could have perhaps done the work in that API to support PATCH, but we chose instead to have a first class specific API for this common operation.